lib/Adapter/TolerantParser/Fixer/DocblockIndentationFixer.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Phpactor\CodeBuilder\Adapter\TolerantParser\Fixer;
+
+use Microsoft\PhpParser\Node;
+use Microsoft\PhpParser\Node\MethodDeclaration;
+use Microsoft\PhpParser\Node\PropertyDeclaration;
+use Microsoft\PhpParser\Node\SourceFileNode;
+use Microsoft\PhpParser\Node\Statement\ClassDeclaration;
+use Microsoft\PhpParser\Parser;
+use Phpactor\CodeBuilder\Domain\TextEdit;
+use Phpactor\CodeBuilder\Domain\StyleProposer;
+use Phpactor\CodeBuilder\Domain\TextEdits;
+use Phpactor\CodeBuilder\Util\TextFormat;
+use Phpactor\CodeBuilder\Util\TextUtil;
+
+class DocblockIndentationFixer implements StyleProposer
+{
+    /**
+     * @var Parser
+     */
+    private $parser;
+
+    /**
+     * @var string
+     */
+    private $newLineChar;
+
+    public function __construct(Parser $parser, string $newLineChar = "\n")
+    {
+        $this->parser = $parser;
+        $this->newLineChar = $newLineChar;
+    }
+
+    public function propose(string $text): TextEdits
+    {
+        $edits = [];
+        $node = $this->parser->parseSourceFile($text);
+
+        foreach ($this->docblockNodes($node) as $docblockNode) {
+            $edits[] = $this->indentationEdits($docblockNode);
+        }
+
+        return TextEdits::fromTextEdits($edits);
+    }
+
+    private function indentationEdits(Node $node): TextEdit
+    {
+        $newLines = [];
+        $baseIndent = '';
+        $level = 0;
+        $lines = TextUtil::lines($node->getLeadingCommentAndWhitespaceText());
+
+        foreach ($lines as $line) {
+            if (preg_match('{^\s*\*}', $line)) {
+                $line = TextFormat::indentationRemove($line);
+                $line = $baseIndent . TextFormat::indentApply($line, ' ', $level);
```

Comment:
Isn't it a bit overkill to use `TextFormat` for that given that `$level` actually has a constant value of 1 which will just append the space ?
```suggestion
                $line = $baseIndent .' '. $line;
```

Reply:
Fixed (and remove the `$level`